kong.conf.default

```diff
@@ -1020,7 +1020,7 @@
                                      # each upstream request to open a new
                                      # connection.
 
-#upstream_keepalive_max_requests = 1000  # Sets the default maximum number of
+#upstream_keepalive_max_requests = 10000 # Sets the default maximum number of
```

Comment:
See the actual value here: https://github.com/Kong/kong/blob/e613aa1cbf3bde1ee0676c3c5be65221c3fdd54e/kong/templates/kong_defaults.lua#L73